From 1c99083e9b82c3d9a7802bff11df6420396ad5db Mon Sep 17 00:00:00 2001
From: Rong Dian <b38775@freescale.com>
Date: Thu, 12 Jul 2012 11:20:17 +0800
Subject: ENGR00216946: battery: increase update period to 2 minutes

increase update period to 2 minutes. Due to improper hardware design,
when enable HDCP function, the I2C2 bus pins function is change to DDC
function,the CPU loading is high when I2C failed transfer data via I2C
bus,so decrease battery update voltage frequency.

Signed-off-by: Rong Dian <b38775@freescale.com>
---
 drivers/power/sabresd_battery.c |   56 +++++++++++++++++++--------------------
 1 file changed, 28 insertions(+), 28 deletions(-)

diff --git a/drivers/power/sabresd_battery.c b/drivers/power/sabresd_battery.c
index 1e13ebf..0e748cd 100755
--- a/drivers/power/sabresd_battery.c
+++ b/drivers/power/sabresd_battery.c
@@ -29,7 +29,7 @@
 #include <linux/sort.h>
 
 
-#define	BATTERY_UPDATE_INTERVAL	5 /*seconds*/
+#define	BATTERY_UPDATE_INTERVAL	120 /*seconds*/
 #define LOW_VOLT_THRESHOLD	2800000
 #define HIGH_VOLT_THRESHOLD	4200000
 #define ADC_SAMPLE_COUNT	6
@@ -70,33 +70,33 @@ static int offset_discharger;
 static int offset_charger;
 
 static battery_capacity chargingTable[] = {
-    {4105,  99},
-    {4100,  98},
-    {4095,  97},
-    {4085,  96},
-    {4075,  95},
-    {4060,  94},
-    {4045,  93},
-    {4030,  92},
-    {4015,  91},
-    {4000,  90},
-    {3900,  85},
-    {3790,  80},
-    {3760,  75},
-    {3730,  70},
-    {3700,  65},
-    {3680,  60},
-    {3660,  55},
-    {3640,  50},
-    {3600,  45},
-    {3550,  40},
-    {3510,  35},
-    {3450,  30},
-    {3310,  25},
-    {3240,  20},
-    {3180,  15},
-    {3030,  10},
-    {2820,  5},
+    {4060,  99},
+    {4035,  98},
+    {4010,  97},
+    {3980,  96},
+    {3950,  95},
+    {3920,  94},
+    {3890,  93},
+    {3860,  92},
+    {3830,  91},
+    {3780,  90},
+    {3750,  85},
+    {3690,  80},
+    {3660,  75},
+    {3630,  70},
+    {3600,  65},
+    {3580,  60},
+    {3550,  55},
+    {3500,  50},
+    {3450,  45},
+    {3400,  40},
+    {3350,  35},
+    {3300,  30},
+    {3250,  25},
+    {3200,  20},
+    {3000,  15},
+    {2900,  10},
+    {2850,  5},
     {2800,  0},
     {0,  0}
 };
-- 
1.7.9.5

